crates/lib/src/executable.rs

```diff
@@ -643,29 +621,18 @@ pub enum Error {
     Compilation(String),
     /// There was a problem when translating the Quil program.
     #[error("There was a problem translating the Quil program: {0}")]
-    Translation(String),
+    Translation(GrpcClientError),
```

Comment:
Many of the other errors look to be handled within `QpuApiError` now, which also has it's own variant with the same type `QpuApiError::GrpcClientError(#[from] GrpcClientError)`